### ISSUE TITLE
Working offline and on-prem

### ISSUE THREAD

**Issue by vayoa** [eoda-dev/py-maplibregl#106]
I'm working in a closed network with no outside access, I took a look at the code and all the js dependencies seem to load through a cdn which means I can't use this library.
Is there any solution to this? Can we have all the dependencies (also the deck.gl ones) baked into the library itself (like plotly does with maplibre)?

**Comment by crazycapivara** [comment]
@vayoa Yes, this would be possible. I did it this way to keep the js bundle size as minimal as possible. Do you use Shiny or the ipywidget?

**Comment by vayoa** [comment]
I use both ipywidgets and streamlit

**Comment by vayoa** [comment]
So how would this work?

**Comment by crazycapivara** [comment]
@vayoa I need to change some stuff in the code.

**Comment by crazycapivara** [comment]
#138

**Comment by crazycapivara** [comment]
@vayoa In the next release `maplibre.ipywidget` comes with all dependencies bundled. For shiny (and streamlit) `maplibre-gl-js` is also included (bundled), but for the moment other dependencies not. But it is planned very soon as well.
